extensions/ql-vscode/src/view/results/ResultTablesHeader.tsx

```diff
@@ -122,17 +128,9 @@ export function ResultTablesHeader(props: Props) {
       </button>
       <div className={tableHeaderItemClassName}>{queryName}</div>
       <div className={tableHeaderItemClassName}>
-        {/*
-            eslint-disable-next-line
-            jsx-a11y/anchor-is-valid
-          */}
-        <a
-          href="#"
-          onClick={openQueryHandler}
-          className="vscode-codeql__result-table-location-link"
-        >
+        <OpenQueryLink onClick={openQueryHandler}>
```

Comment:
The styling on this link seems to have changed.

Before:
![Screenshot 2023-08-17 at 15 14 15](https://github.com/github/vscode-codeql/assets/3749000/2ea420dd-5fe4-49b8-9c96-ef9f39d55443)

After:
![Screenshot 2023-08-17 at 15 07 28](https://github.com/github/vscode-codeql/assets/3749000/576546be-a1fb-4bef-b839-7792dfea9fcb)

Reply:
Thanks for noticing this, I've changed the styling of that component to use styled-components exclusively which fixes the issue.